manageiq-gems-pending.gemspec

```diff
@@ -50,7 +50,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency "net-sftp",                "~>2.1.2"
   s.add_runtime_dependency "nokogiri",                "~>1.7.1"
   s.add_runtime_dependency "openscap",                "~>0.4.3"
-  s.add_runtime_dependency "ovirt",                   "~>0.15.1"
+  s.add_runtime_dependency "ovirt",                   "~>0.16.0"
```

Comment:
I would like to drop the dependency, but there is code here that still depends on it.  Until then, we we have a dependency conflict where we have to merge 2 repos at the same time to keep Travis green.